player/models.py

```diff
@@ -196,9 +201,18 @@ def toggle_shuffle(self, to_active):
             self.save()
             if not self.current_music:
                 self.play_next()
+            else:
+                message = {
+                    'action': 'update',
+                }
+                self.send_message(message)
         else:
             self.shuffle = False
             self.save()
+            message = {
```

Comment:
I think we can do the same action in a full pythonic way....
Je t'en parle demain moins bourré